src/hooks/useClientIntegration.ts

```diff
@@ -0,0 +1,25 @@
+import { useConfigStore } from '#src/stores/ConfigStore';
+
+export enum ClientIntegrations {
+  INPLAYER = 'inplayer',
+  CLEENG = 'cleeng',
+}
+
+const useClientIntegration = () => {
+  const {
+    config: { integrations },
+  } = useConfigStore.getState();
+
+  const isInPlayer = !!integrations?.inplayer?.clientId;
+  const client = isInPlayer ? ClientIntegrations.INPLAYER : ClientIntegrations.CLEENG;
+  const clientId = isInPlayer ? integrations?.inplayer?.clientId : integrations?.cleeng?.id;
+
+  // TODO: More data will follow. Example: access fees will be fetched and returned once user is auth.
```

Comment:
I'm not sure about this TODO.  This hook seems ok as a way to tell components which service we're using, but other logic should probably still be in the controllers and stores.  This logic could also go in a store, so that it's not recalculated every call as well.

Reply:
Yes, the TODO will be removed. The access fees will be created as a service.